### DISCUSSION TITLE
Добавление номера цикла в шаблон печати

### DISCUSSION THREAD

**Discussion by Qb1cc** [totumonline/totum-ru-issues-and-discussions#80] [Как это сделать?] [answered]
Как сюда добавить номер цикла?
![image](https://user-images.githubusercontent.com/84081305/154937954-9a4a8ffe-b761-43d2-ad58-d4100175a503.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Обновите — в текущий [.38 +] добавили, что бы стандартный шаблон печати подтягивал параметр `tabletitle` функции `setTableFormat` из форматирования таблицы.

Вот [видео про эту фигню](https://docs.totum.online/formatting-the-whole-table) — там где-то в середине про модификацию заголовка по условиям.

Будет что-то типа:

```
f1=: setTableFormat(tabletitle: str`"Заказ" ++ $#nci`)
```
